docs/source/filter.md

```diff
@@ -0,0 +1,31 @@
+# AMDirT filter
+
+## Downloading the sequencing data of selected libraries
+
+AMDirT provides three different methods to download the sequencing data of selected libraries from public archives: 
+
+  - direct download from the FTP server using curl
+  - direct download via the FASP protocol using ASPERA
+  - indirect download via the Nextflow pipeline nf-core/fetchngs
+
+### Downloading via curl
+
+### Downloading via the FASP protocol using ASPERA
+
+[FASP](https://en.wikipedia.org/wiki/Fast_and_Secure_Protocol) is a specific protocol that allows the download of large data files at a speed that is usually much higher than when downloading from the FTP server.
+
+Prior to be able to download via this method, make sure that you have the ASPERA connect installed on your system (using `which ascp`). If this is not the case, please refer to this [installation guide](https://www.ibm.com/docs/en/aspera-connect/4.1?topic=suc-installation#installation__section_zfj_wpq_ghb) and download the binary from [here](https://www.ibm.com/aspera/connect/).
+
+AMDirT will return a script that for each sequencing file looks like this following the recommendation from [ENA](https://ena-docs.readthedocs.io/en/latest/retrieval/file-download.html#using-aspera):
+
+`ascp -QT -l 300m -P 33001 -i path/to/aspera/installation/etc/asperaweb_id_dsa.openssh era-fasp@fasp.sra.ebi.ac.uk:path/to/sequencing/file local/target/directory`
```

Comment:
```suggestion
```bash
ascp -QT -l 300m -P 33001 -i path/to/aspera/installation/etc/asperaweb_id_dsa.openssh era-fasp@fasp.sra.ebi.ac.uk:path/to/sequencing/file local/target/directory
```
```